docs/src/API/ParameterDistributions.md

```diff
@@ -8,13 +8,14 @@ CurrentModule = EnsembleKalmanProcesses.ParameterDistributions
 Parameterized
 Samples
 Constraint
+ParameterDistribution
 no_constraint
 bounded_below
 bounded_above
 bounded
-len
+length(c::CType) where {CType <: ConstraintType}
+size(c::CType) where {CType <: ConstraintType}
```

Comment:
Is this required because the methods are from `Base`? I am trying to understand the difference between these and `mean`, `var`, etc, which expand upon StatsBase.

Reply:
yeah docs gave an error that there was duplication in the `length` and `size` when i didn't have the explicit method.

Reply:
I wonder if it doesnt give an error with the clash in `StatsBase` because this package is not a dependence in docs `Project.toml` (if this is what you are surprised about)